cpp/modmesh/onedim/Euler1DCore.cpp

```diff
@@ -111,6 +117,7 @@ void Euler1DCore::update_cfl(bool odd_plane)
 
 void Euler1DCore::march_half_so0(bool odd_plane)
```

Comment:
Could you please also add the profiling hooks to the first-order derivative calculating functions (templates)?

Reply:
Will do!

Reply:
The gist has been appended in the original description.

Reply:
Completed and updated this PR.